README.md

```diff
@@ -78,23 +84,24 @@ class A, that is declared inside a method M, that finally is declared inside a c
 will not count in class C, but only in method M2 (first-level method it is embodied),
 and anonymous class A (first-level class it is embodied).
 
-- *Number of unique words*. Number of unique words in the source code. At method level, it only uses the method body as input. At class level,
+- *Number of unique words*: Number of unique words in the source code. At method level, it only uses the method body as input. At class level,
 it uses the entire body of the class as metrics.
 The algorithm basically counts the number of words in a method/class, after removing Java keywords. Names are split based on camel case and underline (e.g., longName_likeThis becomes four words).
 See `WordCounter` class for details on the implementation.
 
-- *Number of Log Statements*. Number of log statements in the source code. The counting is based on the following regex:
+- *Number of Log Statements*: Number of log statements in the source code. The counting is based on the following regex:
 `line.matches(".*\\.(info|warn|debug|error)\\(.*") || line.matches(".*log(ger)?\\..*");`.
 See `NumberOfLogStatements.java` for more info.
 
-- *Has Javadoc*. Boolean indicating whether a method has javadoc. (Only at method-level for now) 
+- *Has Javadoc*: Boolean indicating whether a method has javadoc. (Only at method-level for now) 
 
-- *modifiers*. public/abstract/private/protected/native modifiers of classes/methods. Can be decoded using `org.eclipse.jdt.core.dom.Modifier`.  
+- *modifiers*: public/abstract/private/protected/native modifiers of classes/methods. Can be decoded using `org.eclipse.jdt.core.dom.Modifier`.  
 
-- *Usage of each variable*. How much each variable was used inside each method.
+- *Usage of each variable*: How often each variable was used inside each method.
 
-- *Usage of each field*. How much each field was used inside each method.
+- *Usage of each field*: How often each local field was used inside each method, also indirect local field accesses are detected.
```

Comment:
maybe a short example of what a _local field access_ is?

Reply:
Done.